src/legacy/core_plugins/region_map/public/choropleth_layer.js

```diff
@@ -78,10 +68,10 @@ export default class ChoroplethLayer extends KibanaMapLayer {
   }
 
 
-  constructor(name, attribution, format, showAllShapes, meta, layerConfig) {
+  constructor(name, attribution, format, showAllShapes, meta, layerConfig, serviceSettings) {
 
     super();
-
+    this.serviceSettings = serviceSettings;
```

Comment:
use `_` prefix for naming consistency (private var)

Reply:
Done